### DISCUSSION TITLE
[unofficial] What do folks want most from CacheLib?

### DISCUSSION THREAD

**Discussion by wonglkd** [facebook/CacheLib#294] [Polls]
I'm not from Meta, but use CacheLib in my research and wonder what fellow users (and prospective users) want most from the library. Poll options are in no particular order, and based on past GitHub issues.

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
